examples/power_spectra_example.py

```diff
@@ -0,0 +1,41 @@
+"""
+===============================
+Power Spectrum of a TimeSeries
+===============================
+
+An example showing how to estimate the power spectrum of a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+
+import sunpy.timeseries
+from sunpy.data.sample import GOES_XRS_TIMESERIES
+
+###############################################################################
+# Let's first load a TimeSeries from sample data.
+
+ts = sunpy.timeseries.TimeSeries(GOES_XRS_TIMESERIES)
+# Data is taken from GOES X-Ray Sensor(XRS), which observes full-disk-integrated
+# solar flux in two broadband channels: 1--8 angstrom (long); and
+# 0.5--4 angstrom (short)
```

Comment:
This should be moved above the python code. 

But what I wanted here was not what is GOES, it was can we use signal analysis on this data. Is it evenly spaced in time, whats its time step, are there any problems that might mean that signal analysis would be tricky etc. We don't want to give the impression you can just singal analysis any signal.

Reply:
Data is not completely evenly sampled,it has 2 outliers .Should I mention this?